rdr_service/dao/bq_participant_summary_dao.py

```diff
@@ -1267,28 +1273,31 @@ def datetime_to_date(val):
         data['distinct_visits'] = len(dates)
         return data
 
-    def _calculate_test_participant(self, summary):
+    def _check_for_test_credentials(self, summary):
         """
-        Calculate if this participant is a test participant or not.
+        Check if this participant is a test participant or not based on email or phone number values
+        that are only supposed to be used for test participant creation.  Note:  test participant status
+        is primarily determined by checking RDR participant table fields (is_ghost_id, is_test_participant, or HPO
+        pairing to TEST) which is done in _prep_participant().  This method is only called if it was not already
+        determined by those primary indicators that the participant is a test participant.
         :param summary: summary data
         :return: dict
         """
-        test_participant = summary['is_ghost_id']
-
-        # Check for @example.com in email address
-        if not test_participant:
-            if summary.get('test_participant') == 1:
-                test_participant = 1
-            # Check to see if the participant is in the Test HPO.
```

Comment:
Why wouldn't we keep this block in?  These are checks to see if the participant has already been marked Test Participant in RDR.

Reply:
The checks for ghost, test participant, or HPO pairing were moved into `_prep_participant()` and I felt it was redundant/more confusing to have them in both places.   So I made this specific to the additional email/phone checks.

This method no longer gets called automatically.   It only gets called if the summary data already created by `_prep_participant()` has not already flagged the participant as test.